### PR TITLE
Allow configuration of Apache HTTP MaxRequestsPerChild and MaxKeepAliveRequests

### PR DESCRIPTION
**Pull request by jchristi**
Apache HTTP configuration values for `MaxRequestsPerChild` and `MaxKeepAliveRequests` are hardcoded to `4000` and `100` respectively. My team has a need to significantly lower the `MaxRequestsPerChild` value to combat memory leakage issues. I made `MaxKeepAliveRequests` also configurable while I was at it so everything in `50-mpm-tuning.conf` is now configurable via environment variables. The default values have been preserved so there should be zero impact to current users. README files have been updated to add the new config options.

### CODE REVIEW

File: 7.2/s2i/bin/run [hunk 1]
```diff
@@ -36,6 +36,8 @@ envsubst < /opt/app-root/etc/php.d/10-opcache.ini.template > ${PHP_SYSCONF_PATH}
 
 export HTTPD_START_SERVERS=${HTTPD_START_SERVERS:-8}
 export HTTPD_MAX_SPARE_SERVERS=$((HTTPD_START_SERVERS+10))
+export HTTPD_MAX_REQUESTS_PER_CHILD=${HTTP_MAX_REQUESTS_PER_CHILD:-4000}
```
Comment (phracek) [inline 7.2/s2i/bin/run:39]:
```suggestion
export HTTPD_MAX_REQUESTS_PER_CHILD=${HTTPD_MAX_REQUESTS_PER_CHILD:-4000}
```

File: 7.3/s2i/bin/run [hunk 2]
```diff
@@ -36,6 +36,8 @@ envsubst < /opt/app-root/etc/php.d/10-opcache.ini.template > ${PHP_SYSCONF_PATH}
 
 export HTTPD_START_SERVERS=${HTTPD_START_SERVERS:-8}
 export HTTPD_MAX_SPARE_SERVERS=$((HTTPD_START_SERVERS+10))
+export HTTPD_MAX_REQUESTS_PER_CHILD=${HTTP_MAX_REQUESTS_PER_CHILD:-4000}
```
Comment (phracek) [inline 7.3/s2i/bin/run:39]:
```suggestion
export HTTPD_MAX_REQUESTS_PER_CHILD=${HTTPD_MAX_REQUESTS_PER_CHILD:-4000}
```

File: 7.4/s2i/bin/run [hunk 3]
```diff
@@ -36,6 +36,8 @@ envsubst < /opt/app-root/etc/php.d/10-opcache.ini.template > ${PHP_SYSCONF_PATH}
 
 export HTTPD_START_SERVERS=${HTTPD_START_SERVERS:-8}
 export HTTPD_MAX_SPARE_SERVERS=$((HTTPD_START_SERVERS+10))
+export HTTPD_MAX_REQUESTS_PER_CHILD=${HTTP_MAX_REQUESTS_PER_CHILD:-4000}
```
Comment (phracek) [inline 7.4/s2i/bin/run:39]:
```suggestion
export HTTPD_MAX_REQUESTS_PER_CHILD=${HTTPD_MAX_REQUESTS_PER_CHILD:-4000}
```

**Review by phracek — CHANGES_REQUESTED** [review]
Hi jchristi, thanks for the PR! But version 7.2,7.3 and 7.4 are currently only supported. The rest is deprecated. Can you please rebase your PR with only supported versions?

**Comment by jchristi** [comment]
Done, suggestions applied and removed changes to old php version images.

**Comment by jchristi** [comment]
The two failing tests seem to be CI infrastructure-related (permissions issues?)

**Comment by phracek** [comment]
[test-openshift][test]

**Comment by phracek** [comment]
Issue with OpenShift 3 is here: https://github.com/sclorg/s2i-php-container/issues/328

**Pull request merged**